src/runtime/TemporalObject.h

```diff
@@ -95,10 +99,12 @@ class TemporalObject : public Temporal {
     static std::map<TemporalObject::DateTimeUnits, int> parseTemporalDurationString(ExecutionState& state, const std::string& isoString);
     static TemporalObject::DateTime parseTemporalYearMonthString(ExecutionState& state, const std::string& isoString);
     static TemporalObject::DateTime parseTemporalMonthDayString(ExecutionState& state, const std::string& isoString);
+    static TemporalObject::TimeZone* parseTemporalTimeZoneString(ExecutionState& state, const std::string& isoString);
```

Comment:
These 3 static functions are declared but I cannot find their definition.

Reply:
parseTemporalYearMonthString - `TemporalObject.cpp` - 497
parseTemporalMonthDayString - `TemporalObject.cpp` - 591
parseTemporalTimeZoneString - `TemporalObject.cpp` - 517

Reply:
I got it. I mentioned these functions because they all return `DateTime` or `TimeZone` which need to be deallocated carefully.